### ISSUE TITLE
Non-existent middleware should be reported

### ISSUE THREAD

**Issue by denishaskin** [balderdashy/sails#3094]
(Not sure if this is a sails or express issue.)

I added a new middleware by modifying `https.js` and adding an item to the `module.exports.http.middleware.order` list.  That middleware wasn't defined anywhere.  It would have saved a lot of debugging time if sails reported that I had specified middleware that it couldn't load.

**Comment by sailsbot** [comment]
Thanks for posting, @denishaskin.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
